website/src/components/CustomBlog/Connectwithus.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
 import styles from "./styles.module.scss";
 import clsx from "clsx";
-import { Github, Twitter, Slack } from "../Icons";
+import { GitHub, Twitter, Slack } from "../Icons";
```

Comment:
This involves a case change on a rename. They can be dangerous on case insensitive file systems. Happy to drop if that's undesirable